src/Extensibility/nunit.extensibility/ExtensionManager.cs

```diff
@@ -95,12 +91,7 @@ public virtual void FindExtensionPoints(params Assembly[] targetAssemblies)
                 foreach (ExtensionPointAttribute attr in assembly.GetCustomAttributes(typeof(ExtensionPointAttribute), false))
                 {
                     if (_extensionPointIndex.ContainsKey(attr.Path))
-                    {
-                        string msg = string.Format(
-                            "The Path {0} is already in use for another extension point.",
-                            attr.Path);
-                        throw new NUnitEngineException(msg);
-                    }
+                        throw new Exception("The Path {attr.Path} is already in use for another extension point.");
```

Comment:
You forgot the leading `$` to make this a formattable string.

```suggestion
                        throw new Exception($"The Path {attr.Path} is already in use for another extension point.");
```

Note that MS doesn't like code throwing `Exception`.
It will raise [CA2201](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca2201).
So we should either create an NUnitExtensibilityException or throw something like `InvalidOperation` or `Argument` or `NotSupport` exception.

Reply:
Fixed the error. I'll go through this assembly and look at all `throw Exception` instances. NUnitExtensibilityException is good for a lot of cases, I think.